bazel/linux/defs.bzl

```diff
@@ -94,8 +137,12 @@ To create a .tar.gz suitable for this rule, you can use the kbuild tool, availab
             doc = "The name of the downloaded kernel. Format is 'distribution-kernel-version-arch', like debian-5.9.0-rc6-rt-amd64.",
             mandatory = True,
         ),
-        "url": attr.string(
-            doc = "The url to download the package from.",
+        "path": attr.string(
```

Comment:
I'd like to retain the ability to fetch by url. Retaining that ability will also allow us a smoother transition, as we can merge your PR in enkit, then update internal, then update the rules to use astore without ever needing atomic commits, and with ease of roll back.

Suggestion:
1) Keep the url attribute here.
2) In the implementation above, just do something like 'if not ctx.attr.url: ...'

(you can use fail() to error out if both astore path and url are provided, or neither of them)